src/main/java/appeng/helpers/DualityInterface.java

```diff
@@ -702,7 +702,8 @@ else if( itemStack.getStackSize() < 0 )
 				long diff = toStore.getStackSize();
 
 				// make sure strange things didn't happen...
-				final ItemStack canExtract = adaptor.simulateRemove( (int) diff, toStore.getItemStack(), null );
+				// TODO: check if OK
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)